Userland/Games/BrickGame/BrickGame.cpp

```diff
@@ -431,6 +483,18 @@ void BrickGame::reset()
     update();
 }
 
+void BrickGame::toggle_pause()
+{
+    // TODO: Investigate weather the timer needs to be paused in someway also
```

Comment:
I'm not an expert in BrickGame, but sounds like this is pretty essential to a pause feature!

Reply:
The timer only determined when it would run an update so it did not interfere with pausing but I've updated the code to include stopping the timer in-case someone wants to do something relying on said timer in future.